server/QuestionsAndAnswers/questions.js

```diff
@@ -0,0 +1,56 @@
+var routes = require('express').Router();
+var axios = require('axios');
+var url = require('url');
+var config = require('../../config');
+var endPoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-atx/qa/questions';
+
+var auth = { headers: { 'Authorization': `${config}` }};
+
+routes.get('/', (req, res) => {
+  axios.get(endPoint, { headers: { 'Authorization': `${config}` }, params: { product_id: req.query.product_id } })
+    .then(response => res.status(200).send(response.data))
+    .catch(err => {
+      console.error(err);
+      res.status(400).end();
+    })
+});
+
+routes.get(`/:question_id/answers`, (req, res) => {
+  axios.get(endPoint + `/${req.params.question_id}/answers`, auth)
+    .then(response => res.status(200).send(response.data))
+    .catch(err => {
+      console.error(err);
+      res.status(400).end();
+    })
+})
+
+routes.post(`/:product_id/question`, (req, res) => {
+  let body = req.body.body, name = req.body.name, email = req.body.email, product_id = req.body.product_id;
```

Comment:
this is prime for destructure!

```node.js
const {
  body,
  name,
  email,
  product_id: productId
} = req.body
```

Reply:
Oh wow, that's so true. I need to get more comfortable with destructure